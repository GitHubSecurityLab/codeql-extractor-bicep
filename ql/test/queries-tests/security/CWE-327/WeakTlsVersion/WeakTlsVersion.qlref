security/CWE-327/WeakTlsVersion.ql