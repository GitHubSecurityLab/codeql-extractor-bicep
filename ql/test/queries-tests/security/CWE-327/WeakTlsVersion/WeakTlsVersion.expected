| app.bicep:18:24:18:28 | String | Weak TLS version detected |
